db/schema.rb

```diff
@@ -41,6 +41,17 @@
     t.datetime "updated_at"
   end
 
+  create_table "preparations", force: :cascade do |t|
+    t.bigint "recipe_id", null: false
```

Comment:
I don't know!

I wrote:

```ruby
t.belongs_to :recipe, null: false, foreign_key: { on_delete: :cascade }
```

in the migration.

I wonder if some version of Rails changed the default... ?